CMakeLists.txt

```diff
@@ -1,180 +1,103 @@
-cmake_minimum_required(VERSION 3.2)
-project(OpenDIS)
-
-## Libraries
-
-
-# create list of DIS6 source files
-file(GLOB DIS6_SOURCES
-  "src/dis6/*.cpp"
-  "src/utils/*.cpp"
-)
-# Define ExampleSender Executable
-add_library(OpenDIS6 SHARED ${DIS6_SOURCES})
-# Add src/ to the include directories to OpenDIS6
-target_include_directories(OpenDIS6 PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src> $<INSTALL_INTERFACE:> )
-# Add compile definition EXPORT_LIBRARY for DIS6 msLibMacro.h
-target_compile_definitions(OpenDIS6 PRIVATE EXPORT_LIBRARY)
-
-# create list of DIS7 source files
-file(GLOB DIS7_SOURCES
-  "src/dis7/*.cpp"
-  "src/utils/DataStream.cpp"
-)
-# Define ExampleSender Executable
-add_library(OpenDIS7 SHARED ${DIS7_SOURCES})
-# Add src/ to the include directories to OpenDIS6
-target_include_directories(OpenDIS7 PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src> $<INSTALL_INTERFACE:> )
-# Add compile definition EXPORT_LIBRARY for DIS7 msLibMacro.h
-target_compile_definitions(OpenDIS7 PRIVATE EXPORT_LIBRARY)
+cmake_minimum_required(VERSION 3.14)
```

Comment:
I just checked the [Travis CI Build] and it only has CMake 3.12.4, so may need to check if that is something you can change in the Travis config. 
If it isn't a config, then another option is to adding a script to build/instal CMake before building OpenDIS.

Reply:
just had a quick look into this, and I think it is just a matter of adding a `dist: focal` line at the top of the .travis.yml.
I'm basing this off looking at the cmake versions on the [Ubuntu packages](https://packages.ubuntu.com/focal/cmake), and clicking though the various releases to find an appropriate version.
... Or I could have just looked that the Travis build Environment pages :facepalm: 
- [Xenial (Default)](https://docs.travis-ci.com/user/reference/focal/#compilers-and-build-toolchain) & [Bionic](https://docs.travis-ci.com/user/reference/bionic/#compilers-and-build-toolchain) : CMake 3.12.4
- [Focal](https://docs.travis-ci.com/user/reference/focal/#compilers-and-build-toolchain) : CMake 3.16.8

Reply:
is there any opposition to using focal?

Reply:
currently set to focal.